nvim/init.lua

```diff
@@ -115,6 +251,17 @@ require('lazy').setup({
     opts = {
     },
   },
+
+  {
+    "folke/noice.nvim",
```

Comment:
Also should be moved to the optional plugins section

Reply:
It can be helpful to have it in this case people can fork and have less merge conflicts. I’m pretty much sure most of the people will add plugins on the top. Our flexibility can be it is in a single file not a separate structure For example NVChad idea is exactly like than In our case is more specific case but having some usability optional features can be a good thing. Most of are looking for IntelliJ or vs code alternative where we are use to with some of the helpers